### PR TITLE
Is monthly cal bug

### PR DESCRIPTION
**Pull request by doutriaux1**
@dnadeau4 ready to review

### CODE REVIEW

**Comment by dnadeau4** [comment]
No test?

**Comment by dnadeau4** [comment]
@doutriaux1 can you write a quick test?  I'd like to see a green light :traffic_light:

File: Lib/times.py [hunk 1]
```diff
@@ -135,12 +135,11 @@ def isMonthly(s):
     units=tim.units
     monthly=1
     for i in range(len(tim)-1):
-      month1=cdtime.reltime(tim[i],units).torel('months since 2000').value
-      month2=cdtime.reltime(tim[i+1],units).torel('months since 2000').value
+      month1=cdtime.reltime(tim[i],units).torel('months since 2000',tim.getCalendar()).value
```
Comment (durack1) [inline Lib/times.py]: @doutriaux1 glad to see that you have fallen into line and are now importing using 3-letter acronyms

Comment (doutriaux1) [inline Lib/times.py]: @durack1 there's no 3 letters import here.

Comment (durack1) [inline Lib/times.py]: @doutriaux1 my mistake, the 3 letter variable `tim=s.getTime()`

**Pull request merged**